tests/lib/rules/no-unescaped-entities.js

```diff
@@ -186,7 +186,20 @@ ruleTester.run('no-unescaped-entities', rule, {
         });
       `,
       parser: 'babel-eslint',
-      errors: [{message: 'HTML entities must be escaped.'}]
+      errors: [{message: '`}` can be escaped with `&#125;`.'}]
+    }, {
+      code: `
+        var Hello = createReactClass({
+          render: function() {
+            return <>foo & bar</>;
+          }
+        });
+      `,
+      parser: 'babel-eslint',
+      errors: [{message: 'HTML entities, \`&\` , must be escaped.'}],
+      options: [{
+        forbid: ['&']
```

Comment:
Can we add a forbid test that use the default parser?

Reply:
@ljharb done :)

Reply:
maybe also tests that cover providing custom alternatives?